addons/gitlab/views.py

```diff
@@ -255,6 +255,7 @@ def gitlab_root_folder(*args, **kwargs):
 # Repos #
 #########
 
+# gitlab parameter shadows import statement
```

Comment:
Good eyes! The `gitlab` param would be better named as `node_settings`. I will update this.